yucca/run/run_training.py

```diff
@@ -113,6 +118,17 @@ def main():
     profile = args.profile
     experiment = args.experiment
 
+    split_idx = args.split_idx
+    split_data_ratio = args.split_data_ratio
+    split_data_kfold = args.split_data_kfold
+
+    if split_data_kfold is None and split_data_ratio is None:
+        split_data_kfold = 5
```

Comment:
Does this also need to be added to run_finetune?

Reply:
yes